index.html

```diff
@@ -89,8 +98,8 @@
       <div style="float:right; width: 340px; padding-top: 10px;">
         Connect typed data to a RESTful service.
       </div>
-      <h2>Typed RESTful Connections</h2>
-      <video controls style="max-width: 100%;"
+      <h2><a href="doc/can-connect%7Cconstructor.html">Typed RESTful Connections</a></h2>
```

Comment:
Maybe we write Typed and Restful connections.  Link `Typed` to:

http://connect.canjs.com/doc/can-connect%7Cconstructor.html

And restful to:

http://connect.canjs.com/doc/can-connect%7Cdata-url.html